lib/coolhand/ruby/configuration.rb

```diff
@@ -3,15 +3,22 @@
 module Coolhand
   # Handles all configuration settings for the gem.
   class Configuration
-    attr_accessor :api_key, :environment, :silent, :intercept_addresses
+    attr_accessor :api_key, :environment, :silent
+    attr_reader :intercept_addresses
 
     def initialize
-      @environment = ENV.fetch("COOLHAND_ENV", "production")
-      @api_key = ENV.fetch("COOLHAND_API_KEY", nil)
-      @silent = ENV.fetch("COOLHAND_SILENT", false)
+      # Set defaults
+      @environment = "production"
+      @api_key = nil
+      @silent = false
+      @intercept_addresses = ["api.openai.com", "api.anthropic.com"]
+    end
```

Comment:
@YarikST let me know if you have any concerns with this? I think it's better, for various reasons, to just use a config block to set the coolhand variables. Relying on specific env vars can cause to many other unexpected headaches.